src/utils/navigation.ts

```diff
@@ -95,6 +95,11 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.OVERSEAS_ENTITY_REVIEW_URL,
     nextPage: []
```

Comment:
Next page is needed.

Reply:
This code has been here for two months

To my knowledge, it is not to do with the ticket and is meant to be like this.

config.UPDATE_CHECK_YOUR_ANSWERS_URL has no page as its previous page

Reply:
Barry is fixing these things but not sure I like asking him ...

Reply:
Do outside this review.